src/Warning.js

```diff
@@ -0,0 +1,14 @@
+class Warning extends Error {
+  constructor (warning) {
+    super()
+
+    const { text, line, column } = warning
+
+    this.name = 'LoaderWarning'
```

Comment:
If you using OOP for `Error` please use `name` as class name - `Warning`. It is best practice.

Reply:
I change it